modules/ml/src/main/java/org/apache/ignite/ml/Model.java

```diff
@@ -32,4 +32,11 @@
     default <X, W> Model<T, X> combine(Model<T, W> other, BiFunction<V, W, X> combiner) {
         return v -> combiner.apply(apply(v), other.apply(v));
     }
+
+    /**
+     * @param pretty Use pretty mode.
+     */
+    default public String toString(boolean pretty) {
```

Comment:
Seems like pretty can be a part of configuration (logging config).

Reply:
yes I understand
but pretty-parameter affect an internal logic of model visualization that knows nothing about logging
I wanted to implement logic of model representation as unification of ModelFormat but I and @ybabak decided to do this later and implement visualization through ToString-method